docs/proposals/ValueSemantics.rst

```diff
@@ -287,10 +287,12 @@ distinction.  In fact both of the following must be present in order
 to observe a difference in behavior:
 
 1. Use of (one of) the big three operations on an object ``x``,
-   creating shared mutable state iff ``x`` is a reference
+   creating shared mutable state [iff] ``x`` is a reference
 
 2. In-place mutation of ``x`` *while a (reference) copy is extant* and
-   thus can be observed through the copy iff ``x`` is a reference.
+   thus can be observed through the copy [iff] ``x`` is a reference.
+
+[iff]: https://en.wikipedia.org/wiki/If_and_only_if "If And Only If"
```

Comment:
I don't think markdown link syntax is going to work in a .rst file.  Please rebuild the docs to confirm that you at least have valid syntax.